library/core/src/slice/mod.rs

```diff
@@ -3434,6 +3436,87 @@ impl<T> [T] {
         }
     }
 
+    /// Split a slice into a prefix, a middle of aligned simd types, and a suffix.
+    ///
+    /// This is a safe wrapper around [`slice::align_to`], so has the same weak
+    /// preconditions as that method.  Notably, you must not assume any particular
+    /// split between the three parts: it's legal for the middle slice to be
+    /// empty even if the input slice is longer than `3 * LANES`.
```

Comment:
Since this wrapper is safe, this should clarify that this is an invalid assumption for `unsafe` code, as the function can no longer be relied upon to maintain invariants in the same way. Safe could would be "merely" logically incorrect.

Reply:
Oh, I wrote "preconditions" here when I meant "postconditions" 🤦